openlibrary/templates/type/author/edit.html

```diff
@@ -106,6 +106,11 @@ <h1>$_("Edit Author")</h1>
                                 <textarea name="author--alternate_names" id="alternate_names" rows="6">$"; ".join(page.alternate_names)</textarea>
                             </div>
                         </div>
+                        <div class="label"><label for="identifiers-display">Identifiers</label></div>
```

Comment:
I believe a label can only be used to annotate an input-y field, not a div.
```suggestion
                        <div class="label">Identifiers</div>
```